stdlib/multiprocessing/popen_fork.pyi

```diff
@@ -0,0 +1,22 @@
+import sys
+
+from .process import BaseProcess
+from .util import Finalize
+
+if sys.platform != "win32":
+    __all__ = ["Popen"]
+
+    class Popen:
+        finalizer: Finalize | None
+        method: str
+        pid: int | None
+        returncode: int | None
+        sentinel: int | None
```

Comment:
Same here, why the `| None`? It's set to `parent_r` in `_launch()`:  https://github.com/python/cpython/blob/bceb197947bbaebb11e01195bdce4f240fdf9332/Lib/multiprocessing/popen_fork.py#L79

And `parent_r` is the first element of the two-element tuple returned by `os.pipe()`: https://github.com/python/cpython/blob/bceb197947bbaebb11e01195bdce4f240fdf9332/Lib/multiprocessing/popen_fork.py#L64

 The typeshed stub says `os.pipe()` returns `tuple[int, int]`: https://github.com/python/typeshed/blob/177bbf7864b544880e7db8f898d14aa5c3561f07/stdlib/os/__init__.pyi#L603

```suggestion
        sentinel: int
```

Reply:
This was similar to the reason explained above, but for `sentinel` it only is set when the pid already exists, so this was modelled with an `| None` to capture the risk of not having the attribute in all circumstances. I don't think it's the right modelling of it, but I wasn't sure how to capture that